tests/test_gsblobstore.py

```diff
@@ -39,5 +39,15 @@ def test_get_checksum(self):
                 self.test_fixtures_bucket,
                 "test_good_source_data_DOES_NOT_EXIST")
 
+    def test_check_bucket_exists(self):
```

Comment:
The test in tests/test_s3blobstore.py should be moved to blobstore_common_tests.  Then it'll be run against both stores.

Reply:
done